JASP-Engine/JASP/R/descriptives.R

```diff
@@ -195,6 +195,29 @@ Descriptives <- function(jaspResults, dataset, options) {
     }
     .descriptivesScatterPlots(jaspResults[["scatterPlots"]], dataset.factors, variables, splitName, options)
   }
+  
+  # Dot plots
+  if(options[['descriptivesDotPlot']]){
+    
+    if(is.null(jaspResults[["DotPlots"]])) {
+      jaspResults[["DotPlots"]] <- createJaspContainer(gettext("Dot Plots"))
+      jaspResults[["DotPlots"]]$dependOn(c("plotVariables", "splitby", "descriptivesDotPlot"))
+      jaspResults[["DotPlots"]]$position <- 11
+    }
+    
+    dotPlots <- jaspResults[["DotPlots"]]
+    
+    for (var in variables) {
+      if(is.null(dotPlots[[var]])) {
+        if (makeSplit) {
+          dotPlots[[var]] <- .descriptivesDotPlots(dataset = splitDat.factors, options = options, variable = var)
+        } else {
+          dotPlots[[var]] <- .descriptivesDotPlots(dataset = dataset.factors, options = options, variable = var)
+          
+        }
```

Comment:
so this is perfectly fine, but you could also write:
```r
dotPlots[[var]] <- .descriptivesDotPlots(
  dataset  = if (makeSplit) splitDat.factors else dataset.factors, 
  options  = options, 
  variable = var
)
```
That way it is ever so slightly more clear that the if statement only affects the first argument.